templates/components/products/ratings.html

```diff
@@ -1,5 +1,9 @@
 <span role="img"
-      aria-label="{{lang 'products.reviews.rating_aria_label' current_rating=rating max_rating=5}}"
+    {{#if product_name}}
+      aria-label="{{lang 'products.reviews.rating_aria_label' rating_target=rating_target current_rating=rating max_rating=5}}"
```

Comment:
do we have cases when product name can be undefined?

Reply:
deprecated var name, fixed